mkt/abuse/serializers.py

```diff
@@ -52,3 +54,13 @@ class WebsiteAbuseSerializer(BaseAbuseSerializer):
 
     class Meta(BaseAbuseSerializer.Meta):
         fields = BaseAbuseSerializer.Meta.fields + ('website',)
+
+
+class ExtensionAbuseSerializer(BaseAbuseSerializer):
+    extension = SplitField(
+        SlugOrPrimaryKeyRelatedField(source='extension', slug_field='slug',
+                                     queryset=Extension.objects.all()),
```

Comment:
You probably want `Extension.objects.without_deleted().public()` as the queryset instead. I'm surprised this isn't done like this for apps/websites too.